pkg/openstack/loadbalancer.go

```diff
@@ -2220,47 +2236,98 @@ func (lbaas *LbaasV2) ensureAndUpdateOctaviaSecurityGroup(clusterName string, ap
 		etherType = rules.EtherType6
 	}
 
+	existingRules, err := getSecurityGroupRules(lbaas.network, rules.ListOpts{SecGroupID: lbSecGroupID})
+	if err != nil && !cpoerrors.IsNotFound(err) {
```

Comment:
I copied that IsNotFound clause here, but I'm fairly sure it's a past mistake here.